### PR TITLE
User-defined API load testing benchmarks [RN35880] [DCP206612]

### PR DESCRIPTION
**Pull request by HansSolos**
server load and client load benchmarks when loading 2500 API tokens and 2500 API definitions.

### CODE REVIEW

File: user-guide/Reference/Metrics/user-defined_API_benchmarks.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+uid: user-defined_API_benchmarks
+---
+
+# User-defined API benchmarks
+
+## Specifications of the test VM server 1
```
Comment (MariekeGO) [inline user-guide/Reference/Metrics/user-defined_API_benchmarks.md:7]: Not sure why a "1" is added if only one server is mentioned. I'd suggest to keep it simple:

```suggestion
## Specifications of the test VM server
```

File: user-guide/Reference/Metrics/user-defined_API_benchmarks.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Intel Core i7-12700
+- 8 cores
+- 32GB RAM
+- HDD
+- Windows 10
+
+## Specifications of the test client PC 1
```
Comment (MariekeGO) [inline user-guide/Reference/Metrics/user-defined_API_benchmarks.md:15]:
```suggestion
## Specifications of the test client
```

File: user-guide/Reference/Metrics/user-defined_API_benchmarks.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+- 12 cores
+- 32GB DDR-5 RAM
+- SSD
+- Windows 11
+
+## Benchmarks
+
+| \# | Specification | Scope | Metric | Configuration |
+| -- | ------------- | ----- | ------ | ------------- |
+| 1 | Server call duration of 2500 API tokens and 2500 API definitions from server 1 to client 1 | DMA | 2s - 2.5s | Clean DMA, no other data |
```
Comment (MariekeGO) [inline user-guide/Reference/Metrics/user-defined_API_benchmarks.md:27]: The current phrasing is a bit unclear. Is this a correct interpretation?

```suggestion
| 1 | Time for the server to retrieve 2500 API tokens and 2500 API definitions | DMA | 2s - 2.5s | Clean DMA without other data |
```

File: user-guide/Reference/Metrics/user-defined_API_benchmarks.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+| 2 | Load testing of 2500 API tokens and 2500 API definitions in DataMiner Cube of client 1 | Cube | 3s - 3.5s | Clean DMA, no other data |
```
Comment (MariekeGO) [inline user-guide/Reference/Metrics/user-defined_API_benchmarks.md:28]:
```suggestion
| 2 | Time for the client to load 2500 API tokens and 2500 API definitions in DataMiner Cube | Cube | 3s - 3.5s | Clean DMA without other data |
```

**Review by MariekeGO — CHANGES_REQUESTED** [review]

File: user-guide/Reference/Metrics/user-defined_API_benchmarks.md (hunk 2 above)
Comment (HansSolos) [reply user-guide/Reference/Metrics/user-defined_API_benchmarks.md:15]: Done

File: user-guide/Reference/Metrics/user-defined_API_benchmarks.md (hunk 4 above)
Comment (HansSolos) [reply user-guide/Reference/Metrics/user-defined_API_benchmarks.md:28]: Done

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**